frontend/plugins/acm/console-extensions.ts

```diff
@@ -218,6 +218,62 @@ const identitiesRoute: EncodedExtension<RoutePage> = {
   },
 }
 
+// ACM resource route extensions - for fleetresourcelink integration
+
+// Managed cluster resource route
+const managedClusterResourceRoute: EncodedExtension = {
+  type: 'acm.resource/route',
+  properties: {
+    model: { group: 'cluster.open-cluster-management.io', kind: 'ManagedCluster', version: 'v1' },
+    handler: { $codeRef: 'resource-routes.acmResourceRouteHandler' },
+  },
+}
+
+// Cluster resource route
+const clusterResourceRoute: EncodedExtension = {
+  type: 'acm.resource/route',
+  properties: {
+    model: { kind: 'Cluster' },
+    handler: { $codeRef: 'resource-routes.acmResourceRouteHandler' },
+  },
+}
```

Comment:
Cluster is not a real resource - it's more like an alias of ManagedCluster that search provides. Please remove this case here and in the handler, and we'll need to handle it as a special case when we convert the search results code to use these extensions.